grumblr/app/assets/javascripts/backbone/views/grumblView.js

```diff
@@ -0,0 +1,25 @@
+var GrumblView = Backbone.View.extend({
+  className: 'grumble',
+  tagName: 'div',
+
+  events: {
+    "click button": "deleteGrumbl"
+  },
+
+  initialize: function() {
+    console.log("created grumbl view");
+    this.listenTo(this.model, 'change', this.render);
+    this.listenTo(this.model, 'destroy', this.render);
```

Comment:
I think this should run `this.remove` when the model is destroyed. Re-rendering probably won't do what you'd expect.